### PR TITLE
Refactor inventories

### PR DESCRIPTION
**Pull request by eroussy**
- Move examples inventories in `examples/inventories`
- Clean examples inventories to contain only necessary variables, the
  rest of the variables will be explained in the README.
- Remove iptables example, it will be explained in the README.
- Remove demo cluster example, as it will be confusing to new users.
- Update the README

### CODE REVIEW

File: inventories/README.md [hunk 1]
```diff
@@ -7,82 +7,43 @@
...
-	* `seapath_vm_definition_example.yml`: virtual machines
+It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup.
+In this directory, we have separated the SEAPATH inventories into :
+* seapath-cluster.yaml : Describe the physical machines in a cluster
+* seapath-standalone.yaml : Describe one standalone physical machine
+* seapath_ovs.yml : Describe an extended network over the cluster (use only in cluster mode)
+* seapath-vm-deployement.yaml : Describe the variable to deploy a VM, either in standalone or cluster mode
 
-The directory also contains the firewall rules `iptables_rules_example.txt` which are referring in `seapath_cluster_definition_example.yml`.
+All these inventories contain only the minimal variables required to configure and run SEAPATH. For advanced configuration, please refer to the wiki page : TODO put link
```
Comment (lucbeaufils) [inline inventories/README.md:21]: is it normal : TODO put link ?

File: inventories/README.md [hunk 2]
```diff
@@ -7,82 +7,43 @@
...
 
-The simple SEAPATH cluster inventory require only minor tweak to be adapted. In all other files described, all the options which do not match a realistic case and need to be modified are indicated.
+### Cluster inventory
 
-### Optional variables
+This inventory describes a cluster using two hypervisors and one observer. It is possible to use a three-hypervisor setup by adding a machine into the `hypervisors` section and removing the `observers` part.
+TODO : Put link to cluster architecture page on wiki when written
```
Comment (lucbeaufils) [inline inventories/README.md:26]: same question : TODO

File: inventories/examples/seapath-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,184 @@
+# This inventory describes a SEAPATH cluster with two hypervisors and one observer.
+# It contains all required variables to configure and run your cluster.
+# Replace all the TODOs to fit your physical machines.
+# More informations on SEAPATH Wiki : TODO put link
```
Comment (lucbeaufils) [inline inventories/examples/seapath-cluster.yaml:4]: wiki link

**Review by lucbeaufils — CHANGES_REQUESTED** [review]

File: inventories/README.md [hunk 4]
```diff
@@ -7,82 +7,43 @@
...
+* seapath-cluster.yaml : Describe the physical machines in a cluster
+* seapath-standalone.yaml : Describe one standalone physical machine
+* seapath_ovs.yml : Describe an extended network over the cluster (use only in cluster mode)
+* seapath-vm-deployement.yaml : Describe the variable to deploy a VM, either in standalone or cluster mode
 
-The directory also contains the firewall rules `iptables_rules_example.txt` which are referring in `seapath_cluster_definition_example.yml`.
+All these inventories contain only the minimal variables required to configure and run SEAPATH. For advanced configuration, please refer to the wiki page : TODO put link
 
-The simple SEAPATH cluster inventory require only minor tweak to be adapted. In all other files described, all the options which do not match a realistic case and need to be modified are indicated.
+### Cluster inventory
```
Comment (dupremathieu) [inline inventories/README.md:23]: Maybe you can change the title to

### seapath-cluster.yaml

or 

### Cluster inventory: seapath-cluster.yaml

To refer to the file.

File: inventories/README.md [hunk 5]
```diff
@@ -7,82 +7,43 @@
...
 
-### Optional variables
+This inventory describes a cluster using two hypervisors and one observer. It is possible to use a three-hypervisor setup by adding a machine into the `hypervisors` section and removing the `observers` part.
+TODO : Put link to cluster architecture page on wiki when written
 
-You can find concrete examples of the variables in the inventories of this directory. Below are optional advanced variables that are not described in the examples.
+### OVS inventory
```
Comment (dupremathieu) [inline inventories/README.md:28]: Same remark.

File: inventories/README.md [hunk 6]
```diff
@@ -7,82 +7,43 @@
...
 
-All inventories examples are in YAML format. The example inventories are:
-* `seapath_demo_cluster_definition_example.yml`: SEAPATH cluster demo
-* `seapath_standalone_definition_example.yml`: SEAPATH standalone hypervisor setup
-* The complex SEAPATH cluster inventories are:
-	* `seapath_cluster_definition_example.yml`: physical machines
-	* `seapath_ovstopology_definition_example.yml`: cluster internal network (Open vSwitch topology)
-	* `seapath_vm_definition_example.yml`: virtual machines
+It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup.
+In this directory, we have separated the SEAPATH inventories into :
```
Comment (dupremathieu) [inline inventories/README.md:15]: This is strange, you described the files twice, here and after in dedicated sections. I suggest removing this part.

File: inventories/README.md [hunk 7]
```diff
@@ -7,82 +7,43 @@
...
-### Optional variables
+This inventory describes a cluster using two hypervisors and one observer. It is possible to use a three-hypervisor setup by adding a machine into the `hypervisors` section and removing the `observers` part.
+TODO : Put link to cluster architecture page on wiki when written
 
-You can find concrete examples of the variables in the inventories of this directory. Below are optional advanced variables that are not described in the examples.
+### OVS inventory
 
-#### Network implementation
+This inventory describes a network Open vSwitch bridge over the cluster. It allows VM to communicate even after a migration toward another hypervisor.
+This work is done through the variable `ovs_bridges`, that is passed to the `seapath-config_ovs` systemd service. This service then calls [python3-setup-ovs](https://github.com/seapath/python3-setup-ovs)
```
Comment (dupremathieu) [inline inventories/README.md:31]: I am not sure mentioning seapath-config_ovs is a good idea. It's adds confusion.

File: inventories/README.md [hunk 8]
```diff
@@ -7,82 +7,43 @@
...
-remove_all_network_config: true
-# If set to true, the network playbook will start by wiping the /etc/netplan/ directory content, this can help cleaning old conflicting files.
-# THIS MUST NOT BE USED WITH skip_recreate_team0_config at the same time or the cluster network config won't be recreated.
-network_simple: true
-# Do not apply the default network configuration. Only team0 will create. The “wired” connexion will not be configured. The bridges br0 and br0vlan will not be created.
-```
+Note: The same extended network bridge is defined for all hypervisors through the variable `cluster_machines`.
 
-### SEAPATH demo cluster inventory
+### Virtual machine inventory
```
Comment (dupremathieu) [inline inventories/README.md:35]: Same remark.

File: inventories/examples/seapath-standalone.yaml [hunk 9]
```diff
@@ -0,0 +1,65 @@
+# This inventory describes a standalone SEAPATH machine (no cluster)
+# It contains all required variables to configure and run your standalone machine.
+# Replace all the TODOs to fit your physical machines.
+# More informations on SEAPATH Wiki : TODO put link
```
Comment (dupremathieu) [inline inventories/examples/seapath-standalone.yaml:4]: Do not forget to put the link.

File: inventories/examples/seapath-vm-deployement.yaml [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+# cluster or standalone machine.
+# It gives two examples, the first on a critical (real-time) VM and the second
+# on a standard VM.
+#
+# Be careful that SEAPATH only handles how to deploy and run the virtual machine.
+# It does not handle any virtual machine inner configuration.
+#
+# Replace all the TODOs to fit your physical machines.
+# More informations on SEAPATH Wiki:
+# - General inventory : TODO put link
```
Comment (dupremathieu) [inline inventories/examples/seapath-vm-deployement.yaml:11]: Do not forget to put the link.

File: inventories/examples/seapath-vm-deployement.yaml [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+# - guest.xml.j2 templated xml file : TODO : put link
```
Comment (dupremathieu) [inline inventories/examples/seapath-vm-deployement.yaml:12]: Do not forget to put the link.

File: inventories/examples/seapath-vm-deployement.yaml [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+      # Configuration part. These variables only work with the guest.xml.j2 template
+      vm_features: [] # features for non real-time VM
+      nb_cpu: 1 # TODO : Number of vCPU of your VM
+      bridges:
+        - name: "br0"
+          mac_address: "52:54:00:e4:ff:03" # TODO
+            # Replace by the mac address you want for your VM
+
+  vars:
+    ansible_user: ansible
```
Comment (dupremathieu) [inline inventories/examples/seapath-vm-deployement.yaml:50]: Only for connection test.

**Review by dupremathieu — CHANGES_REQUESTED** [review]

File: inventories/README.md (hunk 2 above)
Comment (eroussy) [reply inventories/README.md:21]: This wiki page is not written yet, so Yes, this is normal

**Review by lucbeaufils — APPROVED** [review]

**Pull request merged**